thirdparty/DiskANN/src/partition_and_pq.cpp

```diff
@@ -528,17 +521,15 @@ int generate_pq_data_from_pivots(const std::string data_file,
                 block_data_float[index * dim + d] = block_data_tmp[d];
               }
             }
-          }));
+          });
     }
 
```

Comment:
add knowhere::ExecOverBuildThreadPool(tasks); I forgot add it when i added thread pool